src/Components/Blazor/Server/src/Microsoft.AspNetCore.Blazor.Server.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
This cannot target netstandard2.0 because its dependencies now target netcoreapp3.0. This error hasn't appeared until now because the src/Components folder was compiling against a very stale version of its dependencies.

Is converting this to netcoreapp3.0 the right solution here?

```
C:\src\aspnet\AspNetCore\src\Components\Blazor\Server\src\Microsoft.AspNetCore.Blazor.Server.csproj : error NU1201: Project Microsoft.AspNetCore.Hosting.Abstractions is not compatible with netstandard2.0 (.NETStandard,Version=v2.0). Project Microsoft.AspNetCore.Hosting.Abstractions supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0) [C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\CommonExtensions\Microsoft\NuGet\NuGet.targets]
C:\src\aspnet\AspNetCore\src\Components\Blazor\Server\src\Microsoft.AspNetCore.Blazor.Server.csproj : error NU1201: Project Microsoft.AspNetCore.WebSockets is not compatible with netstandard2.0 (.NETStandard,Version=v2.0). Project Microsoft.AspNetCore.WebSockets supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0) [C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\CommonExtensions\Microsoft\NuGet\NuGet.targets]
C:\src\aspnet\AspNetCore\src\Components\Server\src\Microsoft.AspNetCore.Components.Server.csproj : error NU1201: Project Microsoft.AspNetCore.SpaServices.Extensions is not compatible with netstandard2.0 (.NETStandard,Version=v2.0). Project Microsoft.AspNetCore.SpaServices.Extensions supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0) [C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\CommonExtensions\Microsoft\NuGet\NuGet.targets]
C:\src\aspnet\AspNetCore\src\Components\Server\src\Microsoft.AspNetCore.Components.Server.csproj : error NU1201: Project Microsoft.AspNetCore.StaticFiles is not compatible with netstandard2.0 (.NETStandard,Version=v2.0). Project Microsoft.AspNetCore.StaticFiles supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0) [C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\CommonExtensions\Microsoft\NuGet\NuGet.targets]
C:\src\aspnet\AspNetCore\src\Components\Server\src\Microsoft.AspNetCore.Components.Server.csproj : error NU1201: Project Microsoft.AspNetCore.SignalR is not compatible with netstandard2.0 (.NETStandard,Version=v2.0). Project Microsoft.AspNetCore.SignalR supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0) [C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\CommonExtensions\Microsoft\NuGet\NuGet.targets]
```

Reply:
Yes, `netcoreapp3.0` is totally fine for this project.

I expect that changing this will force you to change `Microsoft.AspNetCore.Blazor.Cli` to `netcoreapp3.0` too, which is also good.